src/test/unit/flight_failsafe_unittest.cc

```diff
@@ -39,6 +43,9 @@ extern "C" {
     #include "flight/failsafe.h"
 
     failsafeState_t* failsafeInit(rxConfig_t *intialRxConfig);
+
+    rcControlsConfig_t testRcControlsConfig[MAX_PROFILE_COUNT];
```

Comment:
This should be probably wrapped in macro. Either use PG_REGISTER_PROFILE and then activate profile somehow (probably overkill), or create something like PG_UNITTEST / PG_UNITTEST_ARR PG_UNITTEST_PROFILE (for use when lightweight version is fine).